parse/ConditionParser6.cpp

```diff
@@ -24,176 +22,139 @@ namespace std {
 }
 #endif
 
-namespace {
-    struct condition_parser_rules_6 {
-        condition_parser_rules_6() {
-            const parse::lexer& tok = parse::lexer::instance();
-
-            qi::_1_type _1;
-            qi::_a_type _a;
-            qi::_b_type _b;
-            qi::_val_type _val;
-            qi::eps_type eps;
-            using phoenix::new_;
-            using phoenix::push_back;
-
-            string_ref_vec
-                =   ('[' > +parse::string_value_ref() [ push_back(_val, _1) ] > ']')
-                |    parse::string_value_ref() [ push_back(_val, _1) ]
-                ;
-
-            homeworld
-                =   tok.Homeworld_
-                >   (
-                        (parse::detail::label(Name_token) > string_ref_vec [ _val = new_<Condition::Homeworld>(_1) ])
-                    |    eps [ _val = new_<Condition::Homeworld>() ]
-                    )
-                ;
-
-            building
-                =   (
-                        tok.Building_
-                    >  -(parse::detail::label(Name_token) > string_ref_vec [ _a = _1 ])
-                    )
-                    [ _val = new_<Condition::Building>(_a) ]
-                ;
-
-            species
-                =   tok.Species_
-                >   (
-                        (parse::detail::label(Name_token) > string_ref_vec [ _val = new_<Condition::Species>(_1) ])
-                    |    eps [ _val = new_<Condition::Species>() ]
-                    )
-                ;
-
-            focus_type
-                =   tok.Focus_
-                >   (
-                        (parse::detail::label(Type_token) > string_ref_vec [ _val = new_<Condition::FocusType>(_1) ])
+namespace parse { namespace detail {
+    condition_parser_rules_6::condition_parser_rules_6(
+        const parse::lexer& tok,
+        Labeller& labeller,
+        const parse::detail::condition_parser_grammar& condition_parser,
```

Comment:
`parse::detail::` seems to randomly appear or not before `condition_parser_grammar`